universal-login-react/src/ui/Settings/ManageDevices.tsx

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
+import {DeviceInfo} from '@universal-login/commons';
 
-export const ManageDevices = () => (
+interface ManageDevicesProps {
+  devices?: DeviceInfo[];
+}
+
+export const ManageDevices = ({devices}: ManageDevicesProps) => (
   <div>
-    <button className="settings-btn">Remove this device</button>
-  </div>
+    {
+      devices && devices.map(({os}: DeviceInfo, index: number) => (
+        <div key={index}>
```

Comment:
`index` is probably a bad key, hmmm `publicKey` would be useful afterall :(

Reply:
I will replace it in next PR